src/MOIWrapper.jl

```diff
@@ -374,3 +374,40 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+const statmap = Dict(zip([ 0x00,  0x01, 0x02, 0x03, 0x04, 0x05],
```

Comment:
Constants should be named in upper case: http://www.juliaopt.org/JuMP.jl/v0.19.0/style/#Naming-1.